tests/commands/delete/test_delete_cmd.py

```diff
@@ -98,6 +95,43 @@ def test_delete_variants(mock_app, case_obj, user_obj):
     )
 
 
+def test_delete_omics_variants(mock_app, case_obj, user_obj):
+    """Test the delete variants command's ability to remove omics variants."""
+
+    # GIVEN a database with -omics variants
+    runner = mock_app.test_cli_runner()
+    result = runner.invoke(
+        cli, ["load", "variants", case_obj["_id"], "--outlier", "--rank-treshold", 0]
+    )
+    assert result.exit_code == 0
+    n_initial_vars = sum(1 for _ in store.variant_collection.find())
```

Comment:
You might also assert that the initial count is not 0.

Reply:
Fixed this test now. It had a lot of copy-paste from another one, but this is a different collection than variants!